wgsl/index.bs

```diff
@@ -1808,6 +1810,11 @@ The valid [=enable-extensions=] are listed in the following table.
           [[#subgroup-builtin-functions|subgroup built-in functions]], and
           [[#quad-builtin-functions|quad built-in functions]] in a WGSL module.
           Otherwise, any use will result in a [=shader-creation error=].
+  <tr><td><dfn noexport dfn-for="extension">`primitive_id`</dfn>
+      <td>[[WebGPU#dom-gpufeaturename-primitive-id|"primitive-id"]]
```

Comment:
This doesn't build properly right now. It might need to be done in two steps.

Reply:
Done